src/mpi/init/init.c

```diff
@@ -153,7 +153,9 @@ int MPI_Init(int *argc, char ***argv)
     MPIR_ThreadInfo.isThreaded = 0;
 #endif /* MPICH_IS_THREADED */
 
-    MPIR_T_env_init();
+    mpi_errno = MPIR_T_env_init();
+    if (mpi_errno != MPI_SUCCESS)
```

Comment:
Is this is place to use the new `MPIR_ERR_CHECK` macro?

Reply:
`MPIR_ERR_CHECK` will call `MPIR_ERR_POP`, which will insert an entry into the error chain. In the current code there is such error insertion at `fn_fail` already, therefore, using `MPIR_ERR_POP` or `MPIR_ERR_CHECK` would be redundant.

Reply:
OK, I got it.